### DISCUSSION TITLE
default user no ability to write to file

### DISCUSSION THREAD

**Discussion by Altanis** [microsoft/WSL#9749] [General]
### Windows Version

Microsoft Windows [Version 10.0.19045.2604]

### WSL Version

1.1.3.0

### Are you using WSL 1 or WSL 2?

- [ ] WSL 2
- [X] WSL 1

### Kernel Version

5.15.90.1

### Distro Version

Ubuntu 22.04

### Repro Steps

Install WSL
Create user from startup

### Expected Behavior

`touch filename.txt` should create a file.

### Actual Behavior

Permission is denied. `sudo  touch filename.txt` works, however.

**Comment by Biswa96** [comment]
It depends if the directory has read-write permission to that user.
